### PR TITLE
Task 16.10

### PR DESCRIPTION
**Pull request by AlnurT**

### CODE REVIEW

File: Methods for working with lists/task16_10/task16_10.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+
+def assign_numbers_for_list_symmetry(
+    list_numbers: list, list_new_numbers: list
+) -> tuple:
+    insert_index = len(list_numbers)
+    for num in list_numbers:
+        if list_numbers == list_numbers[::-1]:
+            break
+        list_numbers.insert(insert_index, num)
```
Comment (E1-Bart0) [inline Methods for working with lists/task16_10/task16_10.py:48]: I like this. Very smart

File: Methods for working with lists/task16_10/task16_10.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+Нужно приписать чисел: 4
+Сами числа: [4, 3, 2, 1]
+"""
+
+
+def assign_numbers_for_list_symmetry(
+    list_numbers: list, list_new_numbers: list
+) -> tuple:
+    insert_index = len(list_numbers)
+    for num in list_numbers:
```
Comment (E1-Bart0) [inline Methods for working with lists/task16_10/task16_10.py:45]: It's when you iterate through a list and add elements to the list it's better to iterate through a copy
like for _ in list.copy()

**Review by E1-Bart0 — CHANGES_REQUESTED** [review]

File: Methods for working with lists/task16_10/task16_10.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    return len(list_new_numbers), list_new_numbers
+
+
+def main():
+    print("Введите числа через пробел:", end=" ")
+    list_numbers = list(map(int, input().split()))
+    print(f"\nПоследовательность: {list_numbers}")
+
+    amount_new_numbers = assign_numbers_for_list_symmetry(list_numbers, [])[0]
+    list_new_numbers = assign_numbers_for_list_symmetry(list_numbers, [])[1]
```
[the remark this answers is not in the document]
Comment (AlnurT) [reply Methods for working with lists/task16_10/task16_10.py:59]: 2 times. I`ve already corected.

File: Methods for working with lists/task16_10/task16_10.py (hunk 2 above)
Comment (AlnurT) [reply Methods for working with lists/task16_10/task16_10.py:45]: In order not to damage the list or why?

File: Methods for working with lists/task16_10/test_task16_10.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+def test_symmetry_not_listed():
+    assert assign_numbers_for_list_symmetry([1, 2, 3, 4]) == ([3, 2, 1])
+
+
+def test_partial_symmetry_in_list():
+    assert assign_numbers_for_list_symmetry([1, 3, 2, 2]) == ([3, 1])
+
+
+def test_list_is_already_symmetry():
+    assert assign_numbers_for_list_symmetry([1, 3, 3, 1]) == ([])
```
Comment (E1-Bart0) [inline Methods for working with lists/task16_10/test_task16_10.py:13]: == ([]) Что это значит?

**Review by E1-Bart0 — APPROVED** [review]

**Review by E1-Bart0 — APPROVED** [review]

File: Methods for working with lists/task16_10/test_task16_10.py (hunk 4 above)
Comment (AlnurT) [reply Methods for working with lists/task16_10/test_task16_10.py]: По ошибке оставил скобки

**Pull request merged**